src/documents/views.py

```diff
@@ -261,6 +261,9 @@ def file_response(self, pk, request, disposition):
             file_handle = doc.source_file
             filename = doc.get_public_filename()
             mime_type = doc.mime_type
+            # Support browser previewing csv files by using text mime type
+            if mime_type.find("csv") > -1 and disposition == "inline":
```

Comment:
I think checking `x in {"application/csv",  "text/csv"}` would be a little faster than searching the string, assuming those are the only 2 types

Reply:
Cool